nipype/pipeline/engine/utils.py

```diff
@@ -1021,7 +1021,10 @@ def export_graph(graph_in, base_dir=None, show=False, use_execgraph=False,
                                suffix='.dot',
                                use_ext=False,
                                newpath=base_dir)
-    nx.write_dot(pklgraph, outfname)
+    if float(nx.__version__)<1.11:                           
```

Comment:
Is it necessary to do the version check? NX removed the top-level import, but did they move the function?